PCbuild/pcbuild.sln

```diff
@@ -633,6 +637,38 @@ Global
 		{900342D7-516A-4469-B1AD-59A66E49A25F}.Release|Win32.Build.0 = Release|Win32
 		{900342D7-516A-4469-B1AD-59A66E49A25F}.Release|x64.ActiveCfg = Release|x64
 		{900342D7-516A-4469-B1AD-59A66E49A25F}.Release|x64.Build.0 = Release|x64
+		{4D7C112F-3083-4D9E-9754-9341C14D9B39}.Debug|ARM.ActiveCfg = Debug|ARM
```

Comment:
Do you know why this was reordered? I don't think it matters much but to minimize the diff it might be nice  to restore the ordering (though maybe the ordering was wrong before!)

Reply:
Conflicts resolved